### PR TITLE
Minor cleanup (replace hats, add spaces)

### PR DESCRIPTION
**Pull request by ncave**

### CODE REVIEW

File: src/buildtools/fslex/Lexing.fs [hunk 1]
```diff
@@ -219,7 +219,7 @@ namespace Microsoft.FSharp.Text.Lexing
            LexBuffer<byte>.FromReadFunctions(Some(stream.Read),Some(fun (buf,offset,len) -> stream.AsyncRead(buf,offset=offset,count=len))) 
 
     module GenericImplFragments = 
-        let startInterpret(lexBuffer:LexBuffer<_>)= 
+        let startInterpret(lexBuffer:LexBuffer<_>) =
```
Comment (KevinRansom) [inline src/buildtools/fslex/Lexing.fs]: @ncave 

fslex and fsyacc are not managed in this repo, changes to files in these directories need to be directed to the fslexyacc repo: http://github.com/fsprojects/FsLexYacc 

This is noted in the readme: https://github.com/Microsoft/visualfsharp/blob/master/src/buildtools/README-fslexyacc.md

File: src/buildtools/fsyacc/Lexing.fs [hunk 2]
```diff
@@ -219,7 +219,7 @@ namespace Microsoft.FSharp.Text.Lexing
            LexBuffer<byte>.FromReadFunctions(Some(stream.Read),Some(fun (buf,offset,len) -> stream.AsyncRead(buf,offset=offset,count=len))) 
 
     module GenericImplFragments = 
-        let startInterpret(lexBuffer:LexBuffer<_>)= 
+        let startInterpret(lexBuffer:LexBuffer<_>) =
```
Comment (KevinRansom) [inline src/buildtools/fsyacc/Lexing.fs]: @ncave
fslex and fsyacc are not managed in this repo, changes to files in these directories need to be directed to the fslexyacc repo: http://github.com/fsprojects/FsLexYacc
This is noted in the readme: https://github.com/Microsoft/visualfsharp/blob/master/src/buildtools/README-fslexyacc.md

**Comment by ncave** [comment]
@KevinRansom I "lost" the fslexx and fsyacc changes...

**Comment by KevinRansom** [comment]
Thanks.

File: src/fsharp/LexFilter.fs [hunk 3]
```diff
@@ -2207,7 +2207,7 @@ type LexFilterImpl (lightSyntaxStatus:LightSyntaxStatus, compilingFsLib, lexer,
                   | IEEE32(v)            -> delayMergedToken(IEEE32(if plus then v else -v))
                   | IEEE64(v)            -> delayMergedToken(IEEE64(if plus then v else -v))
                   | DECIMAL(v)           -> delayMergedToken(DECIMAL(if plus then v else System.Decimal.op_UnaryNegation v))
-                  | BIGNUM(v,s)          -> delayMergedToken(BIGNUM((if plus then v else "-"^v),s))
+                  | BIGNUM(v,s)          -> delayMergedToken(BIGNUM((if plus then v else "-"+v),s))
```
Comment (KevinRansom) [inline src/fsharp/LexFilter.fs]: Elsewhere you surrounded '+' with spaces, is there some reason why here you did not?

Comment (ncave) [reply src/fsharp/LexFilter.fs]: No reason, it just looked better. Fixed.

**Pull request merged**